### PR TITLE
[Feature] Add source metadata to key metrics

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description
- Add metadata to profile key metrics

## Screenshot
Screenshot 2022-03-02 at 16 33 51

## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out

### Commits

- [ ]  commits are clean
- [ ]  commit messages are clean

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers
- [ ]  black was run locally (as part of the pre-commit hook)

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: wazimap_ng/profile/serializers/metrics_serializer.py [hunk 1]
```diff
@@ -53,14 +53,16 @@ def MetricsSerializer(profile, geography, version):
         method = algorithms.get(denominator, absolute_value)
         val = method(profile_key_metric, geography)
         if val is not None:
+            metadata = MetaData.objects.filter(dataset=profile_key_metric.variable.dataset).values("source", "description", "url").first()
```
Comment (kilemensi) [inline wazimap_ng/profile/serializers/metrics_serializer.py]: QQ, there is no way of getting metadata when fetching `profile_key_metrics` directly i.e. similar to how they do it [here](https://github.com/OpenUpSA/wazimap-ng/blob/a6353e2cb96c916b568d3e9e6721556f88373af1/wazimap_ng/profile/serializers/utils.py#L32-L34) . Trying to avoid this extra DB call we have to make here.

**Review by kilemensi — APPROVED** [review]
🚀

File: wazimap_ng/profile/serializers/metrics_serializer.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from django.db.models import F
```
Comment (KhadijaMahanga) [inline wazimap_ng/profile/serializers/metrics_serializer.py]:
```suggestion
```
